### ISSUE TITLE
historical.mif: improve data (SE, FE)

### ISSUE THREAD

**Issue by robertpietzcker** [pik-piam/mrremind#441]
The latest historical.mif misses a number of relevant SE/FE variables that should be available from the IEA energy balances:

1. SE|Gases
2. SE|Liquids
3. SE|Heat
4. SE|Solids
5. FE|Industry|Gases
6. FE|Industry|Liquids
7. FE|Industry|Solids

it would be great if they could be added :-)

**Comment by fbenke-pik** [comment]
Have these variables been available before in historical.mif and went missing with recent changes?

**Comment by robertpietzcker** [comment]
For some I think yes, for others only the subcomponents (SE|Gases|Biomass, SE|Gases|Fossil|Coal, SE|Gases|Fossil|Natural Gas) existed, so they would have to be added up

**Comment by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q** [comment]
As of rev6.58:
![image](https://github.com/pik-piam/mrremind/assets/53254462/f3418a84-6204-4c7a-99db-9586986591f5)

**Comment by fbenke-pik** [comment]
Right, the FE|Industry variables mentioned should be available. If you are missing them, it is most likely related to https://github.com/pik-piam/mrremind/issues/442

**Comment by fbenke-pik** [comment]
Right, my bad. For me, the coloring makes it hard to tell the difference for example between FE|Industry|Gases and FE|Industry|Solids|Fossil. So i confused the aggregated variables with the subvariables.


REMIND seems to currently use inputdata revision 6.544, for which i can confirm that the historical.mif is broken. 
Concerning rev6.58: The data seems to be in the historical.mif. For example, I can find FE|Industry|Gases in `rev6.58_62eff8f7_validationremind.tgz` 

![image](https://github.com/pik-piam/mrremind/assets/69258269/521372b7-e724-42fe-994a-e59c53aeb32f)

**Comment by fbenke-pik** [comment]
For SE variables, please check https://github.com/pik-piam/mrremind/pull/443

**Comment by fbenke-pik** [comment]
Closing, as I don't see any immediate todos on my end.
